backend/typescript/services/implementations/camperService.ts

```diff
@@ -127,6 +127,64 @@ class CamperService implements ICamperService {
 
     return camperDtos;
   }
+
+  /* eslint-disable class-methods-use-this */
+  async updateCamperById(
+    camperId: string,
+    camper: UpdateCamperDTO,
+  ): Promise<CamperDTO> {
+    let oldCamper: Camper | null;
+
+    try {
+      oldCamper = await MgCamper.findById(camperId);
+
+      if (oldCamper) {
```

Comment:
```suggestion
      if (camper.camp) {
```
only run these next few checks if there's a new camp to update to